style.css

```diff
@@ -0,0 +1,136 @@
+html {
+  box-sizing: border-box;
+  font-size: 16px;
+  font-family: 'Inter', sans-serif;
+  scroll-behavior: smooth;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  background-color: #fff;
+  margin: 0 auto;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 27px;
+  color: #acb7c3;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+}
+
+.slide {
+  display: block;
+  height: 3px;
+  width: 25px;
+  margin: 5px auto;
+  background-color: #acb7c3;
+}
+
+.content {
+  background-image: url(./images/mobile/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+}
+
+.presentation {
+  margin: 0 27px;
+  text-align: center;
+  padding-top: 65px;
+}
+
+.presentation-text {
+  margin: 15px 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+  line-height: 28px;
+}
+
+.presentation > h2 {
+  padding: 0 3rem;
+}
+
+.title {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  margin-top: 0;
+  margin-bottom: 0;
+  line-height: 54px;
+}
```

Comment:
- As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Use%20responsive%20values%20in%20your%20CSS%20rules%2C%20like%20percentages%20(i.e.%20width%3A%2080%25)%2C%20instead%20of%20pixels), I think it would be better to use responsive Css values instead of pixels. Please Kindly change it. You can use `%`, `vh`, `vw` or even `rem` as they are responsive css values